### PR TITLE
Cherry-picked esignet-related commits from develop for esignet 1.1.0 release

### PR DESCRIPTION
**Pull request by anshulv1401**
Cherry-picked esignet-related commits from develop branch post 13th April, for the release of esignet 1.1.0

- 1e31b94e2ad3b86b8816635d1666173d82d8390c : Updated esignet-mock-plugin version
- 30f8ff3c90f23cb68a4f4f0afcc00fe292f319e1 : MultiLang for login with password added
- f1283fdbc125c7aebc28b0c15d575b6107f1cb17 : [DSD-2764] added esignet-plugins dir and updated Dockerfile
- 8cd83d42be851d8d58148b636143b49a3e034606 : Update artifacts/src/esignet-plugins.
- d16c336473bafb4966a9749f16b370a22172c753 : [DSD-2764] updated esignet-plugins dir and updated Dockerfile (#487)
- dff1907b824122b7a24297934e040a6e9340d4a6 : Update artifacts/src/esignet-plugins/secure-biometric-interface-integ
- 8e370c9b95225c7f5fabb3723996c98c5a029bf6 : transaction time out message added
- 042e6574832d4cbb37bbf1a902da681aa41c757f : Update artifacts/src/esignet-plugins/sign-in-with-esignet/.
- 772f85c0b2bb58f650a436affdceb2ede95f4d6a : ES-176 updated esignet-ida-wrapper version
- ade656545359cdf6b835f7b47a56ac9bd263bb09 : error code invalid qrcode added
- 86e8ed9406c6e18f2f89e377876a30aa686189e6 : Update artifacts/src/esignet-plugins/sign-in-with-esignet/.
- 32943791957c395c4aa1293ccd067298bea4c149 : Update artifacts/src/esignet-plugins/secure-biometric-interface-integ…
- c91c87316a99993c2502b392487490eae5bf18e1 : Update artifacts/src/esignet-plugins/sign-in-with-esignet/.
- 28c8d2f64a93413aa89ae61a26780076c5f79aa9 : Update artifacts/src/esignet-plugins/sign-in-with-esignet/.
- f0e690fb6e56ffa089b2fae88ffbd2889c1e6410 : Update artifacts/src/esignet-plugins/secure-biometric-interface-integ…
- 8a10b547f62f62ab1aff139abacef9102eb0a69b : updated esignet bundle
- d251cdf7d8936564673a50aa37f738c81153e48b : [DSD-3238] updated docker file and esignet plugin dir to mosip plugin

[DSD-2764]: https://mosip.atlassian.net/browse/DSD-2764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DSD-2764]: https://mosip.atlassian.net/browse/DSD-2764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DSD-3238]: https://mosip.atlassian.net/browse/DSD-3238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: artifacts/pom.xml [hunk 1]
```diff
@@ -119,10 +119,10 @@
 		<authentication-wrapper.fileName>authentication-wrapper.jar</authentication-wrapper.fileName>
 		<!-- esignet mock authentication wrapper -->
 		<esignet-wrapper.location>/usr/share/nginx/html/artifactory/libs-release-local/esignet/esignet-wrapper</esignet-wrapper.location>
-		<esignet-mock-wrapper.version>0.9.0</esignet-mock-wrapper.version>
+		<esignet-mock-wrapper.version>0.9.1</esignet-mock-wrapper.version>
 		<esignet-mock-wrapper.fileName>esignet-mock-wrapper.jar</esignet-mock-wrapper.fileName>
 		<!-- esignet IDA wrapper -->
-		<esignet-ida-wrapper.version>1.2.0.1-B3</esignet-ida-wrapper.version>
+		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
```
Comment (vishwa-vyom) [inline artifacts/pom.xml:125]: this the release 1.2.0.1 branch, why we are using 1.2.1-SNAPSHOT ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: artifacts/Dockerfile [hunk 2]
```diff
@@ -118,6 +121,10 @@ COPY /src/deployment/docker/id-authentication/* ${ida_sh_path}/
 
 COPY /src/hsm/client.zip ${hsm_client_path}/
 
+COPY /src/mosip-plugins/sign-in-with-esignet/sign-in-with-esignet.zip ${mosip_plugins_zip_path}/
+
+COPY /src/mosip-plugins/secure-biometric-interface-integrator/secure-biometric-interface-integrator.zip ${mosip_plugins_zip_path}/
```
Comment (vishwa-vyom) [inline artifacts/Dockerfile:126]: As discussed there is no internal service is depending on this, lets not keep it part of the artifactory

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: artifacts/pom.xml (hunk 1 above)
Comment (anshulv1401) [reply artifacts/pom.xml]: corrected the version number to 1.2.0.1-B4

File: artifacts/Dockerfile (hunk 2 above)
Comment (anshulv1401) [reply artifacts/Dockerfile]: Removed the sbi integrator plugin from the artifactory

**Review by ckm007 — APPROVED** [review]

**Review by ase-101 — APPROVED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**